tests/test_app/test_data_store_new.py

```diff
@@ -143,6 +144,38 @@ def full_dstore(write_dir, nc_objects, completed_objects, log_data):
     dstore.write_log(unique_id="scitrack.log", data=log_data)
     return dstore
 
+@pytest.fixture(scope="function")
+def tinydbfile_locked(tmp_dir):
+    path = tmp_dir / "sample_locked.tinydb"
+    shutil.copy(DATA_DIR / path.name, path, )
+    return path
+
+
+@pytest.fixture(scope="function")
+def tinydbfile_notlocked(tmp_dir):
+    try:
+        from fnmatch import fnmatch, translate
+        from tinydb import Query, TinyDB
+        from tinydb.middlewares import CachingMiddleware
+        from tinydb.storages import JSONStorage
+        from cogent3.app.data_store import load_record_from_json
```

Comment:
delete imports not used in this function, e.g. fnmatch, load_record..., DataStoreSq...